cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -514,6 +515,144 @@
     "adata.obs[donor_meta.columns].value_counts()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**See what cell prefixes/suffixes are**\n",
+    "<br>Note: This will need to be adapted depending on what the prefix/suffix looks like</br>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# for suffix\n",
```

Comment:
I think it will be more convenient to collapse prefix & suffix into a single cell. It looks like the only distinction is the `pat`.
So you could do something like this so the user can uncomment whichever pattern is appropriate, and a little easier to see what is going on...

pattern = r'^[AGCT]*' #for suffix
#pattern = r'[ACGT]*-1$' #for prefix

adata_index_split = adata.obs.index.to_series().str.split(pat = pattern, regex=True, expand=True)
adata_index_split.iloc[:,1].value_counts(dropna=False)